src/platform/packages/shared/kbn-esql-ast/src/__tests__/autocomplete.ts

```diff
@@ -157,6 +157,18 @@ export function getFunctionSignaturesByReturnType(
 
   return deduped
     .filter(({ signatures, ignoreAsSuggestion, locationsAvailable }) => {
+      const hasRestrictedSignature = signatures.some((signature) => signature.license);
+      if (hasRestrictedSignature) {
+        const availableSignatures = signatures.filter((signature) => {
+          if (!signature.license) return true;
+          return license === (signature.license.toLocaleLowerCase() as string);
```

Comment:
toLocaleLowerCase() is necessary because I receive capital letters from the signature.license and instead expect lowercase values.